meson.build

```diff
@@ -1194,8 +1194,8 @@ if features['egl-x11']
 endif
 
 plain_gl = get_option('plain-gl').require(
-    get_option('libmpv') and gl_allowed,
-    error_message: 'libmpv was not enabled!',
+    gl_allowed,
+    error_message: 'gl was not enabled!',
 )
 features += {'plain-gl': plain_gl.allowed()}
```

Comment:
I can't actually find anything that depends on plain-gl, can you?

Reply:
plain-gl is supposed to be for compiling libmpv without any platform-specific stuff. So in theory, you would disable everything else, but enable plain-gl and libmpv.

Reply:
But nothing uses `HAVE_PLAIN_GL`, always `HAVE_GL`, so we can drop the plain-gl feature.

Reply:
I went with `features += {'gl': plain_gl.allowed()}`